cmd/sync/sync.go

```diff
@@ -149,11 +150,16 @@ func New() *cobra.Command {
 	}
 
 	cmd.ValidArgsFunction = func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
-		err := root.TryConfigureBundle(cmd, args)
+		ctx := cmd.Context()
+		cmd.SetContext(root.NoPrompt(ctx))
+
+		err := root.MustWorkspaceClient(cmd, args)
 		if err != nil {
 			return nil, cobra.ShellCompDirectiveError
 		}
 
+		cmd.SetContext(ctx)
+
```

Comment:
This can be removed, I think. Why do we need to revert to the original context?

Reply:
@pietern I believe not restoring the context might lead to prompt not appearing when we actually execute the command

Reply:
The shell calls into a new process every time it needs to get new completions, so this won't be an issue.